### PR TITLE
Rancher namespace mover

### PR DESCRIPTION
**Pull request by cikupin**
Github action to move namespace & project mapping on rancher

### CODE REVIEW

File: rancher-ns-mover/action.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: "Rancher Namespacer Mover"
```
Comment (zackijack) [inline rancher-ns-mover/action.yaml]: typo di `Namespacer` tuh @cikupin

File: rancher-ns-mover/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "lint": "eslint src/**/*.ts",
+    "package": "ncc build --source-map",
+    "test": "jest",
+    "all": "yarn run build && yarn run format && yarn run lint && yarn run package"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kitabisa/actions.git"
+  },
+  "keywords": [
```
Comment (zackijack) [inline rancher-ns-mover/package.json:19]: ga ada keyword `action` nih?

**Review by zackijack — DISMISSED** [review]

File: rancher-ns-mover/action.yaml (hunk 1 above)
Comment (cikupin) [reply rancher-ns-mover/action.yaml]: ah iya

File: rancher-ns-mover/package.json (hunk 2 above)
Comment (cikupin) [reply rancher-ns-mover/package.json:19]: gw tambahin

**Review by zackijack — APPROVED** [review]

**Pull request merged**